tox.ini

```diff
@@ -29,6 +29,7 @@ commands = flake8 {posargs}
 [testenv:typing]
 deps =
     mypy~=0.900
+    pytest
```

Comment:
Rather than manually specifying test dependencies, it would be better to tell tox to install all of them via the `tests` extra:

```
[testenv:typing]
extras = tests
...
```

Reply:
Someone please address this.